docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -47,6 +47,7 @@ sudo apt-get install build-essential libz-dev zlib1g-dev
 ----
 xcode-select --install
 ----
+* On Windows you'll need to install the https://visualstudio.microsoft.com/thank-you-downloading-visual-studio/?sku=BuildTools&rel=16[Visual Studio Build Tools]
 ====
```

Comment:
I installed windows 7 sdk based on instructions given from graalvm team info - but that was months ago;  has it changed ?

Reply:
No the official message from the SubstrateVM team has not changed according to the github readme, but according to this [thread](https://github.com/oracle/graal/issues/1850) it looks like the the Windows 7 sdk is only required for older versions of GraalVM and with Java 8. 

It looks like if you want to run JDK 11 on a modern graal you'll need at least VS 2017:

> The JDK 8 versions used Visual Studio 10. The Java team builds JDK 11 with Visual Studio 2017. There is a Windows implementation of GraalVM 19.3 that uses JDK 8. You can try this as a work-around until you get a newer VS version.

So I could update this here to link to VS 2017 and add instructions based on the JDK wanting to be used.

Reply:
@gsmet How would you like this worded?

```
* On Windows you will need to install the Visual Studio Build Tools:
** For JDK 8 based projects you need to install the https://www.microsoft.com/en-us/download/details.aspx?id=8442[Windows 7 SDK]
** For JDK 11 based projects you need to install the https://aka.ms/vs/15/release/vs_buildtools.exe[Visual Studio 2017 Build Tools]
```

Reply:
I would just keep it simple and only describe what needs to be done for JDK 11 projects.

JDK 8 is deprecated in master so no need to make things more awkward than necessary.